docs/source/hardware/lowpriority.rst

```diff
@@ -0,0 +1,75 @@
+
+.. _partition_names:
+
+**********
+Low priority partition
+**********
+:sub:`Updated September, 2022`
+
+
+Introduction
+===================
+``lowpriority`` is a new partition on our high performance computing (HPC) cluster - Discovery, that allows the research community to use private resources on the HPC cluster when they are idle. 
+This new partition has hardware that is not otherwise available to the general research community and, in time, could double the resources available to NURC users. 
+This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
+
+When to use the low priority partition
+===================
+
+From most to least-recommended scenarios:
+
+1. Code that can be `checkpointed <https://rc-docs.northeastern.edu/en/latest/best-practices/checkpointing.html?highlight=array#gromacs-checkpointing-example>`_
+2. Jobs that fit on a single node
+3. Jobs that require multiple nodes (eg, `MPI <https://northeastern-university-rc-public-documentation--19.com.readthedocs.build/en/19/software/mpi.html>`_)
+4. When waiting for your jobs to get started is too hard
+
+How to use the low priority partition
+===================
+
+To use the low priority partition, just add ``lowpriority`` (no space between 'low' and 'priority') to your partition list. 
+The partition list is a comma-delimited list of values indicating the order of preference for assigning the job to a particular partition. 
+Hence, the low priority partition must be specified after a general partition has been mentioned, as ``srun --partition=short,lowpriority``. 
+For the below example, we will exclude partition ``short`` and focus on ``lowpriority``. When you run the ``squeue`` command, you can see 
```

Comment:
For the below example -> For the example below

Reply:
done